app/services/idv/steps/verify_base_step.rb

```diff
@@ -105,6 +105,128 @@ def track_aamva
         doc_auth_log.aamva = true
         doc_auth_log.save!
       end
+
+      def enqueue_job
+        return if flow_session[verify_step_document_capture_session_uuid_key]
+        return invalid_state_response if invalid_state?
+
+        pii[:uuid_prefix] = ServiceProvider.find_by(issuer: sp_session[:issuer])&.app_id
+
+        if pii[:ssn].present?
+          throttle = Throttle.new(
+            target: Pii::Fingerprinter.fingerprint(pii[:ssn]),
+            throttle_type: :proof_ssn,
+          )
+
+          if throttle.throttled_else_increment?
+            @flow.analytics.track_event(
+              Analytics::THROTTLER_RATE_LIMIT_TRIGGERED,
+              throttle_type: :proof_ssn,
+              step_name: self.class,
+            )
+            redirect_to idv_session_errors_ssn_failure_url
+            return
+          end
+        end
+
+        document_capture_session = create_document_capture_session(
+          verify_step_document_capture_session_uuid_key,
+        )
+
+        document_capture_session.requested_at = Time.zone.now
+
+        idv_agent.proof_resolution(
+          document_capture_session,
+          should_proof_state_id: should_use_aamva?(pii),
+          trace_id: amzn_trace_id,
+        )
+      end
+
+      def idv_agent
+        @idv_agent ||= Idv::Agent.new(pii)
+      end
+
+      def invalid_state?
+        pii.blank?
+      end
+
+      def invalid_state_response
+        mark_step_incomplete(:ssn)
+        FormResponse.new(success: false)
+      end
```

Comment:
This is all copy/pasted from the doc auth verify step so that it can be used across doc auth and in-person flows